spec/fixtures/build.gradle.kts

```diff
@@ -1,122 +1,42 @@
 /*
- * Copyright 2014 Google Inc.
```

Comment:
This was a `build.gradle` file that I basically added parentheses to, but then realized that kotlin's differences are much more nuanced, so I swapped the file with a basic boilerplate `build.gradle.kts`.